lib/jwk/keystore.js

```diff
@@ -585,15 +600,16 @@ JWKStore.isKey = function(obj) {
  *
  * @param {Object|String} key The value to coerce into a Key
  * @param {String} [form] The format of a String Key to expect
+ * @param {Object} [extras] extra jwk fields inserted when importing from a non json string (eg "pem")
  * @returns {Promise(JWK.Key)} A promise for the coerced Key.
  */
-JWKStore.asKey = function(key, form) {
+JWKStore.asKey = function(key, form, extras) {
```

Comment:
I realized a short while after add this support that I missed this.  Thanks for adding it!